app/apis/ims/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""
+app.apis.ims.init
+-------------------
+
+"""
+from flask_restful import Api
+
+from apis.ims import caliper, rest, lti_11, lti_13, cim, assignment_grades, deep_link, proctoring, names_roles, groups, jwks, outcomes
+
+
+def evaluate_url(url, prefix_url):
+    """
+
+    :param url:
+    :param prefix_url:
+    :return:
+    """
+    return (prefix_url if prefix_url else '') + url
+
+
+def init_app(api: Api, prefix_url=None):
+    """
+    :param prefix_url:
+    :param api:
+    :return:
+    """
+
+    api.add_resource(caliper.Send, evaluate_url('/caliper/send', prefix_url))
+    api.add_resource(caliper.Register, evaluate_url('/caliper/register', prefix_url))
+    api.add_resource(caliper.Caliper, evaluate_url('/caliper', prefix_url))
+
+    api.add_resource(rest.Auth, evaluate_url('/rest/auth', prefix_url))
+    api.add_resource(rest.User, evaluate_url('/rest/user', prefix_url))
+    api.add_resource(rest.Course, evaluate_url('/rest/course', prefix_url))
+
+    api.add_resource(outcomes.Outcomes, evaluate_url('/lti/outcomes', prefix_url))
+    api.add_resource(outcomes.SendOutcomes, evaluate_url('/lti/send_outcomes', prefix_url))
+    api.add_resource(outcomes.GetOutcomes, evaluate_url('/lti/get_outcomes', prefix_url))
+
+    api.add_resource(lti_11.Membership, evaluate_url('/lti/membership', prefix_url))
+    api.add_resource(lti_11.Lti, evaluate_url('/lti/11/launch', prefix_url))
+
+    api.add_resource(cim.CIMRequest, evaluate_url('/contentItem/request', prefix_url))
+    api.add_resource(cim.ContentItemData, evaluate_url('/contentItem/data', prefix_url))
+
+    api.add_resource(lti_13.Lti13Launch, evaluate_url('/lti/13/launch', prefix_url))
+    api.add_resource(lti_13.JWTPayloadData, evaluate_url('/lti/13/payloadData', prefix_url))
+    api.add_resource(lti_13.Login, evaluate_url('/lti/13/login', prefix_url))
+
+    api.add_resource(deep_link.DeepLinkingPayloadData, evaluate_url('/deepLink/payloadData', prefix_url))
+    api.add_resource(deep_link.DeepLinkingContent, evaluate_url('/deepLink/content', prefix_url))
+    api.add_resource(deep_link.DeepLinkingOptions, evaluate_url('/deepLink/options', prefix_url))
+    api.add_resource(deep_link.DeepLinkingFrame, evaluate_url('/deepLink/frame', prefix_url))
+
+    api.add_resource(proctoring.ProctoringPayloadData, evaluate_url('/proctoring/payloadData', prefix_url))
+    api.add_resource(proctoring.BuildProctoringStartReturnPayload,
+                     evaluate_url('/buildProctoringStartReturnPayload', prefix_url))
+    api.add_resource(proctoring.BuildProctoringEndReturnPayload,
+                     evaluate_url('/buildProctoringEndReturnPayload', prefix_url))
+
```

Comment:
The Proctoring URL pattern is different than the others. Is there a reason you with withe /buildProctoringStartReturnPayload instead of something like /proctoring/payloadData/return/start?